third_party/wabt/src/interp/jit/jit.cc

```diff
@@ -44,6 +48,80 @@ void JITFunction::call() const {
 
   func.func_entry = func_entry_;
   func.code(context, reinterpret_cast<void*>(start));
+
+  std::vector<int32_t> offsets;
```

Comment:
We have an allocator for this. The problem is, if we change the stack allocator, we need to change this as well. Hard to maintain.